Dockerfile

```diff
@@ -48,89 +48,110 @@ RUN update-alternatives --set gcc /usr/bin/gcc-15 \
 RUN gcc --version && gcc-15 --version && gcc-14 --version
 RUN update-alternatives --get-selections
 
-RUN 	git clone https://github.com/hyprwm/hyprwayland-scanner && \
+RUN 	cd /tmp;git clone https://github.com/hyprwm/hyprwayland-scanner && \
```

Comment:
rather than `cd /tmp` each time, we could use a top level `WORKDIR` statement

I'd be tempted to put it somewhere in opt (`/opt/hyprland` that's where I install on my machine anyway?) - so it stays persistent and isn't destroyed after running - since we'll need to run the container once we're done to copy the tarballs out, and it's handy to have all the logs etc for debugging.